charts/argo-cd/Chart.yaml

```diff
@@ -22,3 +22,4 @@ dependencies:
 annotations:
   artifacthub.io/changes: |
     - "[Added]: added ImagePullPolicy to repo server's init containers"
```

Comment:
Could you please remove old changelog as they are aggregated/linked per release. Thanks for your contribution

Reply:
Sure, I removed that line.